### PR TITLE
Cobie Module Refactoring

### PR DESCRIPTION
**Pull request by ihabelaghoury**
This is the preliminary refactoring of COBie module

### CODE REVIEW

File: src/ifcblenderexport/blenderbim/bim/module/cobie/operator.py [hunk 1]
```diff
@@ -0,0 +1,85 @@
+import bpy
+import blenderbim.bim.module.context.add_context as add_context
+import blenderbim.bim.module.context.remove_context as remove_context
```
Comment (Moult) [inline src/ifcblenderexport/blenderbim/bim/module/cobie/operator.py:3]: add_context / remove_context are not needed

File: src/ifcblenderexport/blenderbim/bim/module/cobie/operator.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+from blenderbim.bim.ifc import IfcStore
+from blenderbim.bim.module.context.data import Data
```
Comment (Moult) [inline src/ifcblenderexport/blenderbim/bim/module/cobie/operator.py:5]: This import is not needed.

File: src/ifcblenderexport/blenderbim/bim/module/cobie/ui.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
+from bpy.types import Panel
+from blenderbim.bim.module.context.data import Data
```
Comment (Moult) [inline src/ifcblenderexport/blenderbim/bim/module/cobie/ui.py:2]: Import not needed

**Comment by Moult** [comment]
You will also need to edit https://github.com/IfcOpenShell/IfcOpenShell/blob/v0.6.0/src/ifcblenderexport/blenderbim/bim/__init__.py

Add your module at L23, L306, L352, and L373.

File: src/ifcblenderexport/blenderbim/bim/module/cobie/operator.py (hunk 1 above)
Comment (ihabelaghoury) [reply src/ifcblenderexport/blenderbim/bim/module/cobie/operator.py:3]: Sure, by bad 👍

**Pull request merged**